lib/compiler/aro_translate_c.zig

```diff
@@ -668,6 +898,26 @@ fn transCreateNodeAPInt(c: *Context, int: aro.Value) !ZigNode {
     return res;
 }
 
+fn cIsStruct(ty: Type) bool {
```

Comment:
No need for these - you can use `ty.is(.@"struct")` / `ty.is(.@"union")`

Reply:
Thanks for the heads up, that's good to know. Updated.